src/components/welcome/MyAccounts/index.tsx

```diff
@@ -135,7 +135,7 @@ const AccountsList = ({ safes, onLinkClick, isSidebar = false }: AccountsListPro
           </Box>
 
           {/* All Accounts */}
-          <Accordion sx={{ border: 'none' }}>
+          <Accordion expanded={pinnedSafes.length === 0} sx={{ border: 'none' }}>
```

Comment:
When the user pins their first safe it closes now. We can use `defaultExpanded` to keep it open in that case